client/blocks/eligibility-warnings/index.jsx

```diff
@@ -2,144 +2,100 @@
-		siteId,
-		isEligible,
-		eligibilityData,
-		isPlaceholder,
-	} = props;
+export const EligibilityWarnings = ( {
+	backUrl,
+	eligibilityData,
+	hasBusinessPlan,
+	isEligible,
+	isJetpack,
+	isPlaceholder,
+	onProceed,
+	siteId,
+	siteSlug,
+	translate,
+} ) => {
+	const context = -1 !== backUrl.indexOf( 'plugins' ) ? 'plugins' : 'themes';
 
-	const holdMessages = getHoldMessages( translate );
-	const holds = get( eligibilityData, 'eligibilityHolds', [ 'PLACEHOLDER', 'PLACEHOLDER' ] );
 	const warnings = get( eligibilityData, 'eligibilityWarnings', [] );
 
-	const classes = classNames( {
-		'eligibility-warnings__message': true,
-		'eligibility-warnings__placeholder': isPlaceholder,
-	} );
+	let holds = get( eligibilityData, 'eligibilityHolds', [ 'PLACEHOLDER', 'PLACEHOLDER' ] );
+	const bannerHolds = filter( holds, hold => -1 !== [ 'NO_BUSINESS_PLAN', 'NOT_USING_CUSTOM_DOMAIN' ].indexOf( hold ) );
+	holds = difference( holds, bannerHolds );
+
+	const classes = classNames(
+		'eligibility-warnings',
+		{ 'eligibility-warnings__placeholder': isPlaceholder }
+	);
 
 	return (
-		<div className="eligibility-warnings">
+		<div className={ classes }>
 			<QueryEligibility siteId={ siteId } />
-			<SectionHeader label={ translate( 'Conflicts' ) } />
-
-			{ map( holds, ( error, index ) =>
-				<Card key={ index } className={ classes }>
-					<Gridicon icon="notice" className="eligibility-warnings__error-icon" />
-					<div className="eligibility-warnings__message-content">
-						<div className="eligibility-warnings__message-title">
-							{ translate( 'Error: %(title)s', { args: { title: holdMessages[ error ].title } } ) }
-						</div>
-						<div className="eligibility-warnings__message-description">
-							{ holdMessages[ error ].description }
-						</div>
-					</div>
-					<div className="eligibility-warnings__message-action">
-						<Button href={ holdMessages[ error ].supportUrl } target="_blank" rel="noopener noreferrer">
-							{ translate( 'Resolve' ) }
-						</Button>
-					</div>
-				</Card>
-			) }
-
-			{ map( warnings, ( { name, description, supportUrl }, index ) =>
-				<Card key={ index } className={ classes }>
-					<Gridicon icon="notice" className="eligibility-warnings__warning-icon" />
-					<div className="eligibility-warnings__message-content">
-						<div className="eligibility-warnings__message-title">
-							{ translate( 'Unsupported feature: %(name)s', { args: { name } } ) }
-						</div>
-						<div className="eligibility-warnings__message-description">
-							{ description }
-						</div>
-					</div>
-					<div className="eligibility-warnings__message-action">
-						<a href={ supportUrl } target="_blank" rel="noopener noreferrer">
-							<Gridicon icon="help-outline" className="eligibility-warnings__warning-action" />
-						</a>
-					</div>
+
+			{ 'plugins' === context && ! hasBusinessPlan && ! isJetpack &&
+				<Banner
+					description={ translate( 'Please upgrade to install this plugin.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Upgrade to the Business plan to install plugins.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).